components/elements/Button.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import colors from '../../utils/colors';
+
+const Button = styled('button')`
+  border: none;
+  padding: .6em 1.4em;
+  background-color: ${colors.black};
+  color: ${colors.white};
+  box-shadow: -0.25em 0 0 rgba(255, 255, 255, 0.6), 0.25em 0 0 rgba(255, 255, 255, 0.6), 0 0.25em 0 rgba(255, 255, 255, 0.6), 0 -0.25em 0 rgba(255, 255, 255, 0.6) ;
```

Comment:
Maybe it would be good to break this into multiple lines?

```suggestion
  box-shadow:
    -0.25em 0 0 rgba(255, 255, 255, 0.6),
    0.25em 0 0 rgba(255, 255, 255, 0.6),
    0 0.25em 0 rgba(255, 255, 255, 0.6),
    0 -0.25em 0 rgba(255, 255, 255, 0.6);
```

Reply:
Also it would be good to parametrise the width of the border width